src/core/variable.jl

```diff
@@ -507,8 +507,10 @@ function variable_mc_oltc_transformer_tap(pm::AbstractUnbalancedPowerModel; nw::
 
     if bounded
         for tr_id in p_oltc_ids, p in 1:length(ref(pm,nw,:transformer,tr_id,"f_connections"))
-            set_lower_bound(var(pm, nw)[:tap][tr_id][p], ref(pm, nw, :transformer, tr_id, "tm_lb")[p])
-            set_upper_bound(var(pm, nw)[:tap][tr_id][p], ref(pm, nw, :transformer, tr_id, "tm_ub")[p])
+            if haskey(ref(pm, nw, :transformer, tr_id), "tm_lb") && haskey(ref(pm, nw, :transformer, tr_id), "tm_ub")
```

Comment:
I am not sure why this check is necessary, because in principle tm_lb and tm_ub should always exist in the MATHEMATICAL model. If you can confirm for me that this is broken we can add that as a new bug to be fixed.

Reply:
@pseudocubic I removed the check. The issue with the missing `tm_lb` and `tm_ub` occurred because I had used the transformation `remove_all_bounds!` with the engineering model. Some of the examples in in the examples folder used this transformation, and I was following suit. Although I removed the check, some other user might do the same transform for the same reason and then receive the same error. Perhaps the error message should be extended by adding something like "This error can occur if the transformation `remove_all_bounds!` is used." 

I have copied the file `ieee13_feeder.dss` unchanged from PowerModelsONM and placed it in /test/data/opendss. I added a test for the new function `solve_mc_opf_oltc` in the test/multinetwork.jl file. The testset label is also `solve_mc_opf_oltc`. I'm not sure if you want it in the multinetwork test group, or the opf test group. Also, the test values are simply the values that I obtained when running the function. I don't know if they make physical sense or not.

My pull request should pass all tests now.